Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-with-servicenow-playbook.yml

```diff
@@ -0,0 +1,357 @@
+      version: -1
+      name: IOT_CONFIG list configured?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: 'to check whether the IOT_CONFIG is defined'
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: lists.IOT_CONFIG
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: be71d420-9199-41ce-8cee-31f7db39a642
+    type: condition
+    task:
+      id: be71d420-9199-41ce-8cee-31f7db39a642
+      version: -1
+      name: ServiceNow Table Existed?
+      description: Check if a given value exists in the context. Will return 'no'
+        for empty empty arrays. To be used mostly with DQ and selectors.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "8"
+    scriptarguments:
+      value:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: aca86a67-dc23-4563-833f-1307866b85e1
+    type: regular
+    task:
+      id: aca86a67-dc23-4563-833f-1307866b85e1
+      version: -1
+      name: servicenow-create-record
+      description: Creates a new record in the specified ServiceNow table.
+      script: ServiceNow v2|||servicenow-create-record
+      type: regular
+      iscommand: true
+      brand: ServiceNow v2
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      custom_fields:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.custom_fields}
+      fields:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.fields}
+      table_name:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 275,
+          "y": 1185
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: ServiceNow Table Name
+      output:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    - incidentfield: ServiceNow Record ID
+      output:
+        simple: ${ServiceNow.Record.ID}
+    skipunavailable: false
+    quietmode: 0
+  "9":
+    id: "9"
+    taskid: 849b6d99-d818-4697-8eef-f34dcf7068cc
+    type: regular
+    task:
+      id: 849b6d99-d818-4697-8eef-f34dcf7068cc
+      version: -1
+      name: Manual investigation
```

Comment:
Maybe some follow up task? did the manual investigation have any findings yes/no etc.

Reply:
@liorkol added this manual step according to Lior, any ideas, Lior?